systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -337,6 +338,7 @@ void testJvmAndResources() {
     }
 
     @Test
+    @Tag(SCALABILITY)
     void testKafkaConnectScaleUpScaleDown() {
```

Comment:
I am not sure if those tests should have `SCALABILITY` tag. I will leave the decision to other.

Reply:
I would keep it there, although test scales up just from 1 -> 2 pods. 
Maybe scale to 3~5 and tag is deserved there :)